src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/AtomPub10ServiceDocumentFormatter.cs

```diff
@@ -475,7 +475,7 @@ private ResourceCollectionInfo ReadCollection(XmlReader reader, Workspace worksp
                 {
                     if (reader.IsStartElement(Atom10Constants.TitleTag, Atom10Constants.Atom10Namespace))
                     {
-                        result.Title = Atom10FeedFormatter.ReadTextContentFrom(reader, "//app:service/app:workspace/app:collection/atom:title[@type]", _preserveAttributeExtensions);
+                        result.Title = new Atom10FeedFormatter().ReadTextContentFrom(reader, "//app:service/app:workspace/app:collection/atom:title[@type]", _preserveAttributeExtensions);
                     }
```

Comment:
This just seems wrong. This won't use any custom parsers, is there a custom parser we should be using? As this method is internal, can you convert it back to static and add parameter(s) for the parser(s)?

Reply:
I removed the StringParser. This needs not to be changed anymore.